fluent-plugin-google-cloud.gemspec

```diff
@@ -21,10 +21,10 @@ eos
 
   gem.add_runtime_dependency 'fluentd', '~> 0.10'
   gem.add_runtime_dependency 'googleapis-common-protos', '~> 1.3'
-  gem.add_runtime_dependency 'google-api-client', '~> 0.17'
-  gem.add_runtime_dependency 'google-cloud-logging', '~> 1.3', '>= 1.3.2'
-  gem.add_runtime_dependency 'googleauth', '~> 0.6'
-  gem.add_runtime_dependency 'grpc', '~> 1.0'
+  gem.add_runtime_dependency 'google-api-client', '~> 0.14'
+  gem.add_runtime_dependency 'google-cloud-logging', '~> 1.2', '<= 1.2.3'
+  gem.add_runtime_dependency 'googleauth', '~> 0.5'
```

Comment:
Why do we need to pin `googleauth`? Does that contribute to the leak, or is it not backward-compatible enough for the earlier version of `grpc`?

Reply:
It is not backward-compatible with previous version of `google-cloud-logging`.

Reply:
I'm confused — before #180 we already had `googleauth` at `0.6`. Were we just picking up a new enough version of `google-cloud-logging` that this wasn't a problem?

Reply:
The dependency chains are:

`google-cloud-logging` depends on `google-gax`.
`google-gax` depends on `grpc` and `googleauth`.
`grpc` also depends on `googleauth`.

I just double checked that we need `'googleauth', '~> 0.5'` because:
https://rubygems.org/gems/grpc/versions/1.2.5-universal-darwin.

On a second thought, I think we should just remove the explicit `googleauth` version and let dependencies handle it.